core/container.go

```diff
@@ -319,6 +323,11 @@ func getDockerOptions(c corescan.Context) *backend.ContainerCreateConfig {
 			Source: resultsPath,
 			Target: "/data/results",
 		},
+		{
+			Type:   mount.TypeBind,
+			Source: reportPath,
+			Target: "/data/results/report",
```

Comment:
Please extract this into constant, and use it in https://github.com/JetBrains/qodana-cli/blob/08a2a3743c879c8e332047345e85f4c8a32aba7b/platform/commoncontext/compute.go#L180 aswell so we can understand that these two are linked

Reply:
Alternatively I think you can pass the `--report-dir` to docker, similar to https://github.com/JetBrains/qodana-cli/blob/3d1cf31a93491b41c004626fcdc38b541d76aa37/core/container.go#L57

Reply:
Also I guess it related to other mount constants aswell, would be cool to extract them 🙏

Reply:
awesome idea, liked it a lot and implemented in the last commit